src/analytics.ts

```diff
@@ -62,9 +63,17 @@ export function createAnalyticsInstance(options?: Options) {
                     }
                 })
                 _pending_identify_calls = []
+
+                //    Auto-identify with user_id from core_data if available
+                const existingUserId = core_data?.user_id
+                if (existingUserId && !isUUID(existingUserId)) {
+                    _rudderstack?.identifyEvent(existingUserId, {
+                        language: core_data?.user_language || 'en',
+                    })
+                }
```

Comment:
same here

Reply:
When the page refreshes, the user is still logged in (their user_id is available in the initialization options), but without the auto-identify logic, the user doesn't get identified on page refresh, which breaks tracking.